src/2_session.js

```diff
@@ -57,15 +57,16 @@ session.update = function(storage, newData) {
  */
 session.patch = function(storage, data, updateLocalStorage){
 
-	const merge = (source, patch) =>{
-		return utils.merge(source, patch);
```

Comment:
Is the ` utils.merge` broken? If so, can we not fix the utils function?

Reply:
👍